bite-size-notes/week-2-d-Intro-To-Convex-Optimization.md

```diff
@@ -0,0 +1,143 @@
+# Intro to Convex Optimization
+
+#### Authors note 
+I tried to focus on getting down the more formal definitions for optimization and the notation for writing up problems. Usually, when I go back and try to remember a topic the notation and an explanation of some of the basics I may have forgotten help me to brush of the dust so to speak. I am also quite new to this topic, so some of the insights might seem surface level, but I am open to diving deeper into areas in the revision since I can already tell I've learned a lot just by writing these notes. 
+
+## Scope and Objective
+These notes will serve as a brief introduction to Convex Optimization within the field of Controls. Since this is also my introduction to optimization, it will cover more of the basics of convex optimization and what distinguishes it from other optimization problems.  
+
+## Introduction to optimization
+Optimization is a topic that is used in mathematics, computer science, engineering, and economics. It is a method of finding the best or optimal solution to a defined problem or system. Optimization problems can occur on an incredibly broad scale, going from optimizing everyday things like coffee consumption based on your relaxed constraints in your everyday life to optimizing aircraft performance based on 500 strict mechanical, aerodynamic, and financial constraints. Almost any system can be made into an optimization problem, and due to advancements in mathematical computing, most problems can be solved using a standard method and code base.
```

Comment:
1. remove "going" in the second line. 
2. what does the 500 mean? Like 500 constraints? Probably remove that -- just say you can have aircraft performance constraints such as mechanical , aerodynamic, and financial requirements. 
3. Last sentence change to: "most CONVEX problems can be solved using standard methods and coding packages". I wouldn't say most problems in general i.e., nonconvex problems, can be solved by standard methods. These usually involve relaxations, complex convexification methods, or iterative linearization methods. 
4. Going off the last point, maybe mention that optimization problems are classified into two main broad groups: convex and non-convex. Convex are "easy" and nonconvex are hard. There is a quote by R. T. Rockafeller (a leading scholar in optimization theory and a professor at uw!) said "The great watershed in optimization isn't between linearity and nonlinearity, but convexity and nonconvexity." So mentioning nonconvex problems are hard is a good intro to why we care about convex problems.